src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_operation_factory.py

```diff
@@ -0,0 +1,365 @@
+#---------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#---------------------------------------------------------------------------------------------
+
+## SDK COMMAND OPERATIONS
```

Comment:
Or alternatively:
`cli_command('my command', 'azure.mgmt.compute.operation', 'create_or_update')`
That would be cleaner. I get your point about intellisense and... ... pylint. 😐 (although VSCode seems to go crazy with red highlights around imports anyways)

This is the one optimization I think we should brainstorm on because it changes the way people have to onboard commands and makes it more obtuse (the rest of the changes would not be exposed to command authors).